src/System Application/App/Permissions/SystemTablesBasic.PermissionSet.al

```diff
@@ -65,5 +66,7 @@ permissionset 66 "System Tables - Basic"
                   tabledata "Style Sheet" = RIMD,
                   tabledata "System Object" = Rimd,
                   tabledata "Tenant Profile Page Metadata" = Rimd,
-                  tabledata "Token Cache" = Rimd;
+                  tabledata "Token Cache" = Rimd,
+                  tabledata "Performance Profile Scheduler" = Rimd,
```

Comment:
These would then go away and the new permission set edit should be included instead

Reply:
I added the two new permission sets